modules/rules_helm/0.6.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,47 @@
+"""abrisco/rules_helm"""
+
+module(
+    name = "rules_helm",
+    version = "0.6.0",
```

Comment:
This doesn't match the directory version

Reply:
Done.